### DISCUSSION TITLE
How to edit this "Chips Card"

### DISCUSSION THREAD

**Discussion by lbreggi** [dwainscheeren/dwains-lovelace-dashboard#802] [Q&A]
Hi! Apologies if this question was answer in another discussion, I tried to find the info but i was not luck.
I would like to edd this part of the dashboard
![Screenshot 2024-04-15 at 12 54 00 PM](https://github.com/dwainscheeren/dwains-lovelace-dashboard/assets/25729150/50cb9baa-8b33-4428-8ab9-e9063b580718)

Is is possible to remove or to edit the content?

thank you!

**Comment by ghost** [comment]
That would be nice!
Would like to get ridd of both Hadmin and the devices part of that chip section.

**Comment by trudetta** [comment]
I am also interested in this.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
